src/pages/ReimbursementAccount/PersonalInfo/HelpLinks.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import styles from '@styles/styles';
+import CONST from '@src/CONST';
+
+const propTypes = {
+    /** Style for wrapping View */
+    // eslint-disable-next-line react/forbid-prop-types
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Translate function */
+    translate: PropTypes.func.isRequired,
```

Comment:
Any reason we prefer passing the translate function here instead of using the hook here?